src/api/payment-requests/payment-requests.md

```diff
@@ -713,6 +713,83 @@ Alternatively you can provide an external transaction Id and the Centrapay [Asse
 | 403    | {% break _ REFUND_NOT_SUPPORTED %}          | The asset type does not support refunds.                                                                                                                                                                                                                                    |
 | 403    | {% break _ REFUND_WINDOW_EXCEEDED %}        | The time since the payment exceeds the window of time a payment request can be refunded in.                                                                                                                                                                                 |
 
+<a name="cancel">
+### Cancel a Payment Request **EXPERIMENTAL**
+
+{% reqspec %}
+  POST '/api/payment-requests/{paymentRequestId}/cancel'
+  auth 'jwt'
+  path_param 'paymentRequestId', 'MhocUmpxxmgdHjr7DgKoKw'
+{% endreqspec %}
+
+{% h4 Example response payload %}
+{% json %}
+{
+  "type": "cancelled",
+	"cancellationReason": "CANCELLED_BY_MERCHANT",
+  "value": {
+    "currency": "NZD",
+    "amount": "1000",
+  },
+  "assetType": "centrapay.nzd.main",
+  "paymentRequestId": "MhocUmpxxmgdHjr7DgKoKw",
+  "merchantName": "Centrapay Café",
+  "merchantId": "26d3Cp3rJmbMHnuNJmks2N",
+  "merchantAccountId": "C4QnjXvj8At6SMsEN4LRi9",
+  "merchantConfigId": "5efbe2fb96c08357bb2b9242",
+  "createdAt": "2021-06-08T04:04:27.426Z",
+  "createdBy": "crn::user:0af834c8-1110-11ec-9072-3e22fb52e878",
+  "paymentRequestCreatedBy": "crn::user:0af834c8-1110-11ec-9072-3e22fb52e878",
+  "activityNumber": "2",
+}
+{% endjson %}
+
+{% h4 Error Responses %}
+
+| Status |             Code              |                             Description                              |
+| :----- | :---------------------------- | :------------------------------------------------------------------- |
+| 403    | {% break _ REQUEST_EXPIRED %} | The payment request has expired and therefore cannot be cancelled.   |
+| 403    | {% break _ REQUEST_PAID %}    | The payment request has been paid and therefore cannot be cancelled. |
+
+<a name="void">
+### Void a Payment Request **EXPERIMENTAL**
+
```

Comment:
Do we need to explain that void chooses between cancelling and refunding a payment request?

Reply:
It might be better to use language consistent with what we have in the [guides section](https://docs.centrapay.com/guides/merchant-terminals) of the docs :).

Details aren't as important. This is an example of rest without put so think of it as a business process that we're applying to the payment request.

Reply:
Can we break out void and cancel as separate PRs?

Cancel is easier to talk through :)